bin/jwtwebhook.py

```diff
@@ -26,23 +26,24 @@
 from splunk_helper import data_encryption
 from modular_input import ModularInput, Field, IntegerField, FilePathField
 from webhooks_input_app.flatten import flatten
```

Comment:
nit: No longer used import, we can probably delete the library too...

Reply:
Let me make some changes today. Forgot to clean up.